longestString/longestString.js

```diff
@@ -1,4 +1,37 @@
 /*
  * Write a function that accepts an array of strings.
  * Return the longest string in the array.
- */
\ No newline at end of file
+ */
+
+// 1. make a for loop over an array
+// 2. have a placeholder outside the loop
+// 3. check of the i'th element is larger than the placeholder and replace
+// 4. return the placeholder at the end.
+
+//  const longeststring = (ans) => {
+//   let placeholder = [];
+//    for (let i = 0; i < ans.length, i++;){
```

Comment:
Typo after ans.length you've inserted a comma and not a semi-colon. It'll getcha everytime.